scripts/prepare_lambda_deployment.py

```diff
@@ -34,6 +35,12 @@ def transform_tf(input_json):
         assert 'layers' not in func
         func['layers'] = ["${var.layer_arn}"]
 
+        # Inject runtime ES environment variables provisioned by Terraform
+        es_endpoint = ('${var.azul_es_domain.endpoint}', 443)
+        es_instance_count = '${var.azul_es_domain.cluster_config[0].instance_count}'
```

Comment:
This `.cluster_config[0].instance_count` still irks me as it reveals an intimate dependency on TF resources or data sources and what properties they have. I wonder if it be better to pass the endpoint and instance count as separate variables to the module rather than the entire domain status object as one variable.